pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -229,6 +229,17 @@ func (b *BridgePodInterface) preparePodNetworkInterfaces() error {
 		return err
 	}
 
+	// turn TX offload checksum because it causes dhcp failures
+	if err := EthtoolTXOff(b.podInterfaceName); err != nil {
```

Comment:
Maybe move this just before `startDHCHPServer` function or we also want to disable the tx for l2 interfaces?